src/components/App/App.js

```diff
@@ -55,7 +55,11 @@ function App() {
   }, []);
 
   const loadStatistics = async () => {
-    return await getfarmerStatisticsHandler();
+    try {
+      return await getfarmerStatisticsHandler();
+    } catch (err) {
+      toast.error(err);
```

Comment:
We are passing everywhere to toast ```err.message``` instead of just ```err``` so I guess this should be the same